docs/using/recipes.md

```diff
@@ -523,18 +523,20 @@ Please note that the [Delta Lake](https://delta.io/) packages are only available
 ```dockerfile
 FROM jupyter/pyspark-notebook:latest
 
-ARG DELTA_CORE_VERSION="0.8.0"
+ARG DELTA_CORE_VERSION="1.0.0"
 
 USER root
+RUN pip install delta-spark==${DELTA_CORE_VERSION}
```

Comment:
```suggestion
ARG DELTA_CORE_VERSION="1.0.0"
RUN pip install --quiet --no-cache-dir delta-spark==${DELTA_CORE_VERSION}

USER root
```

If there's no reason to install under root (I don't think there is one), we should install everything from jovyan.

Reply:
Also, let's not cache these files, because they might be quite big and caching will significantly increase docker image size.

Reply:
I agree with @mathbunnyru.

Reply:
Not sure, why we are receiving an error on the checks. I think it is unrelated to the actual pull request as it seems to be missing docker.  

```
Executable `docker` not found
```